Microsoft.DotNet.Interactive/Commands/IKernelCommand.cs

```diff
@@ -5,5 +5,6 @@ namespace Microsoft.DotNet.Interactive.Commands
 {
     public interface IKernelCommand
     {
+        string Name { get; }
```

Comment:
In the past I've tended to avoid this by letting the serialization layer handle it based on the actual `System.Type`. Within the domain layer, we know the instance's type, so this property isn't useful except in the serialization code.